### ISSUE TITLE
no-missing-{import,require} rule not handling index.js file when requiring a directory

### ISSUE THREAD

**Issue by bjrmatos** [mysticatea/eslint-plugin-node#21]
my rules are the following:

```js
"node/no-missing-import": 2,
"node/no-missing-require": 2,
"node/no-unpublished-import": 2,
"node/no-unpublished-require": 2,
"node/no-unsupported-features": [2, {"version": 4}],
"node/shebang": 2
```

i'm requiring two directories `./fileSchemas` and `./utils/logger` both directories have a index.js inside but `eslint-plugin-node` reports an error.

![image](https://cloud.githubusercontent.com/assets/4262050/12690679/3b14464a-c6b3-11e5-8cb1-440f090c9a87.png)

[according to node docs](https://nodejs.org/dist/latest-v4.x/docs/api/modules.html) this is a valid `require` when dealing with directories.

![image](https://cloud.githubusercontent.com/assets/4262050/12690659/1a522968-c6b3-11e5-9bad-647bd47b3808.png)

**Comment by mysticatea** [comment]
Thank you for this issue.

Oh, yes! I have forgotten it.

**Comment by bjrmatos** [comment]
thnk you!
